Governance_v_1_6 (#304)
MinBZK/Algoritmekader: docs/maatregelen/0-org-05-bestaande-governance.md

---
title: Sluit algoritmegovernance aan op bestaande governance
id: urn:nl:ak:mtr:org:05

> toelichting: Onderzoek binnen de organisatie welke governancestructuren er al zijn en hoe algoritmegovernance daarop kan inhaken. Communicatie en samenwerking tussen organisatieonderdelen is doorlopend van groot belang.

Reviewer: ```suggestion
toelichting: Onderzoek binnen de organisatie welke governancestructuren er al zijn en hoe algoritmegovernance daarop kan aansluiten. Communicatie en samenwerking tussen organisatieonderdelen is doorlopend van groot belang.
```

levenscyclus:
- organisatieverantwoordelijkheden

onderwerp:
- governance

rollen:
- projectleider
- beleid-en-advies

---

<!-- tags -->

## Maatregel

> Onderzoek binnen de organisatie welke governancestructuren er al zijn en hoe algortimegovernance daarop kan inhaken. Communicatie en samenwerking tussen organisatieonderdelen is doorlopend van groot belang.

Reviewer: ```suggestion
Onderzoek binnen de organisatie welke governancestructuren er al zijn en hoe algortimegovernance daarop kan inhaken. 
```

## Toelichting

Bij de vormgeving van een algoritmegovernance van een organisatie is het van belang om aansluiting en samenwerking te bewerkstelligen met huidige governancestructuren binnen de organisatie, zoals:

- IT governance
- [datagovernance](../onderwerpen/data#datagovernance-en-datamanagement)
- informatiebeveiliging zoals governance rondom de [NIS2 richtlijn](https://www.digitaleoverheid.nl/overzicht-van-alle-onderwerpen/nis2-richtlijn/)
- [privacygovernance](https://www.cip-overheid.nl/media/eeqkauey/20200814-handleiding-privacy-governance-v3_2.pdf)

In veel organisaties werken privacy- en informatiebeveiliging nauw samen van strategisch organisatieniveau tot operationeel omdat deze onderwerpen raken aan beide domeinen. 
Dit geldt in nog grotere mate voor vraagstukken rondom de inzet van algoritmes, die vragen om samenwerking en expertise vanuit veel verschillende invalshoeken.
Communicatie en vindbaarheid tussen de verschillende domeinen is hierin essentieel. 
De eindverantwoordelijkheid voor algoritmes moet eenduidig worden belegd binnen de organisatie.

### Aandachts- en actiepunten

> * Welke lessen zijn geleerd met de implementatie van de AVG?

Reviewer: ```suggestion
* Welke lessen zijn geleerd met de implementatie van de AVG of de toepassing van de BIO?
```

* Is er iemand intern verantwoordelijk gemaakt voor (toezicht op) algoritmes?

> * Hoe communiceren experts vanuit verschillende onderwerpen zoals privacy, informatiebeveiliging en data, die het onderwerp algoritme raken, op dit moment?

Reviewer: ```suggestion
* Hoe werken experts vanuit verschillende onderwerpen zoals privacy, informatiebeveiliging en data op dit moment samen als het gaat om de inzet van algoritmes? 
```

> Een strategie of visie zoals vastgelegd in een meerjarenbeleidsplan sturing geven en bestuurlijke steun opleveren in het maken van afwegingen.

Reviewer 2: ```suggestion
Zorg voor [bestuurlijke betrokkenheid](0-org-04-politiek-bestuurlijke-verantwoordelijkheid.md) bij het maken van deze afwegingen. Een strategie of visie zoals vastgelegd in een meerjarenbeleidsplan helpt sturing te geven bij het maken van afwegingen.
```

> ## Aandachts- en actiepunten

Reviewer 2: ```suggestion
### Aandachts- en actiepunten
```

* Het [opstellen van RACI-model](2-owp-01-rollen-en-verantwoordelijkheden.md) helpt bij het in kaart brengen van de samenhang tussen verschillende governancestructuren.

> * Hoe worden afwegingen gemaakt in de organisatie tussen innovatie en compliance?

Reviewer: ```suggestion
* Hoe zorgt een organisatie ervoor dat met enige snelheid innovatieve oplossingen kunnen worden ontwikkeld en gebruikt,  en dat tegelijkertijd aan voldoende (basis) vereisten wordt voldaan voor verantwoorde inzet van algoritmes.  
```

> * Een RACI-model kan ook helpen hierbij (zie ook maatregelen over rollen en verantwoordelijkheden).

Reviewer 2: ```suggestion
* Het [opstellen van RACI-model](2-owp-01-rollen-en-verantwoordelijkheden.md) helpt bij het in kaart brengen van de samenhang tussen verschillende governancestructuren.
```

* Een orgaan als een ethische commissie (AI) kan helpen te verbinden en keuzes te maken. Let wel op dat dit niet een 'praatgroep' wordt zonder daadwerkelijk mandaat en draagvlak.

## Voorbeelden
Een concreet voorbeeld van samen optrekken is [deze handreiking](https://www.cip-overheid.nl/media/av0dmahv/20230614-gezamenlijk-gebruik-iama-en-model-dpia-rijksdienst-v1-0.pdf) om de IAMA en DPIA gezamelijk uit te voeren.

!!! example "Voorbeeld: Ervaringen Kadaster" 

>     1) Bij inwerkingtreding van de AVG ging men snel aan de slag met DPIA’s, vinkenlijstjes. Let op voor een schijndossier: Snap wat er toegepast moet worden,    voordat je aan de slag gaat; niet uitvoeren om maar te laten zien dat je eraan voldoet.
>     2) Nadenken over governance. De FG is bij de wet geregeld, maar er is meer nodig. Welke profielen heeft de ondersteuning voor de uitvoering; (centrale) privacy officers en is hier ruimte voor?
>     3) De FG kijkt naar de wet en interpreteert dit. Zorg dat deze interpretatie gelijk is binnen de organisatie en de collega’s die binnen de governance structuur samenwerken.
>     4) Er is beleid gemaakt, maar te weinig bezetting en/of andere prioriteiten. Zorg voor een realistisch beleid en daarop een realistische planning. We willen vaak te veel (de gehele wetgeving meteen beheersen).
>     _Centrale privacy officer Kadaster_

Reviewer 2: ```suggestion

    1. Bij inwerkingtreding van de AVG ging men snel aan de slag met DPIA’s, vinkenlijstjes. Let op voor een schijndossier: Snap wat er toegepast moet worden,    voordat je aan de slag gaat; niet uitvoeren om maar te laten zien dat je eraan voldoet.
    2. Nadenken over governance. De FG is bij de wet geregeld, maar er is meer nodig. Welke profielen heeft de ondersteuning voor de uitvoering; (centrale) privacy officers en is hier ruimte voor?
    3. De FG kijkt naar de wet en interpreteert dit. Zorg dat deze interpretatie gelijk is binnen de organisatie en de collega’s die binnen de governance structuur samenwerken.
    4. Er is beleid gemaakt, maar te weinig bezetting en/of andere prioriteiten. Zorg voor een realistisch beleid en daarop een realistische planning. We willen vaak te veel (de gehele wetgeving meteen beheersen).
    _Centrale privacy officer Kadaster_
```

Onderzoek binnen de organisatie welke governancestructuren er al zijn en hoe algortimegovernance daarop kan inhaken. Communicatie en samenwerking tussen organisatieonderdelen is doorlopend van groot belang.

## Toelichting

Bij de vormgeving van een algoritmegovernance van een organisatie is het van belang om aansluiting en samenwerking te bewerkstelligen met huidige governancestructuren binnen de organisatie, zoals:

- IT governance
- [datagovernance](../onderwerpen/data#datagovernance-en-datamanagement)
- informatiebeveiliging zoals governance rondom de [NIS2 richtlijn](https://www.digitaleoverheid.nl/overzicht-van-alle-onderwerpen/nis2-richtlijn/)
- [privacygovernance](https://www.cip-overheid.nl/media/eeqkauey/20200814-handleiding-privacy-governance-v3_2.pdf)

In veel organisaties werken privacy- en informatiebeveiliging nauw samen van strategisch organisatieniveau tot operationeel omdat deze onderwerpen raken aan beide domeinen. 
Dit geldt in nog grotere mate voor vraagstukken rondom de inzet van algoritmes, die vragen om samenwerking en expertise vanuit veel verschillende invalshoeken.
Communicatie en vindbaarheid tussen de verschillende domeinen is hierin essentieel. 

> De eindverantwoordelijkheid voor algoritmes moet bij één entiteit belegd worden, die let op het betrekken van de juiste (interne) partijen. 

Reviewer: ```suggestion
De eindverantwoordelijkheid voor algoritmes moet eenduidig worden belegd binnen de organisatie.
```

> Bij algoritmegovernance moet rekening worden gehouden met mogelijk conflicterende belangen. 

Reviewer: ```suggestion
```

> Bijvoorbeeld, het kan voorkomen dat de business of IT-teams innovatie willen nastreven, terwijl compliance teams juist vragen om het afremmen van innovatie. 

Reviewer: ```suggestion
```

> Bijvoorbeeld, het kan voorkomen dat de business of IT-teams innovatie willen nastreven, terwijl compliance teams juist vragen om het afremmen van innovatie. 

Reviewer: ```suggestion
```

> Een algortime-eindverantwoordelijke dient hierin de afwegingen te maken, in samenspraak.

Reviewer: ```suggestion
```

> Zorg voor [bestuurlijke betrokkenheid](0-org-04-politiek-bestuurlijke-verantwoordelijkheid.md) bij het maken van deze afwegingen. Een strategie of visie zoals vastgelegd in een meerjarenbeleidsplan helpt sturing te geven bij het maken van afwegingen.

Reviewer: ```suggestion
```

## Aandachts- en actiepunten
* Welke lessen zijn geleerd met de implementatie van de AVG?
* Is er iemand intern verantwoordelijk gemaakt voor (toezicht op) algoritmes?
* Hoe communiceren experts vanuit verschillende onderwerpen zoals privacy, informatiebeveiliging en data, die het onderwerp algoritme raken, op dit moment?
* Het [opstellen van RACI-model](2-owp-01-rollen-en-verantwoordelijkheden.md) helpt bij het in kaart brengen van de samenhang tussen verschillende governancestructuren.
* Hoe worden afwegingen gemaakt in de organisatie tussen innovatie en compliance?

> * Een orgaan als een ethische commissie (AI) kan helpen te verbinden en keuzes te maken. Let wel op dat dit niet een 'praatgroep' wordt zonder daadwerkelijk mandaat en draagvlak.

Reviewer: ```suggestion
* Een orgaan als een ethische adviescommissie kan waardevolle (zwaarwegende) adviezen geven in hoeverre de werking van een algoritme wenselijk of passend is. Daarbij helpt het als deze adviezen concrete aanbevelingen bevatten, zodat ontwikkelteams van algoritmes indien nodig de werking van het algoritme kunnen wijzigen.  
```

## Voorbeelden
Een concreet voorbeeld van samen optrekken is [deze handreiking](https://www.cip-overheid.nl/media/av0dmahv/20230614-gezamenlijk-gebruik-iama-en-model-dpia-rijksdienst-v1-0.pdf) om de IAMA en DPIA gezamelijk uit te voeren.

!!! example "Voorbeeld: Ervaringen Kadaster" 

    1) Bij inwerkingtreding van de AVG ging men snel aan de slag met DPIA’s, vinkenlijstjes. Let op voor een schijndossier: Begrijp wat er toegepast moet worden, voordat je aan de slag gaat; niet uitvoeren om maar te laten zien dat je eraan voldoet.
    2) Nadenken over governance. De FG is bij de wet geregeld, maar er is meer nodig. Welke profielen heeft de ondersteuning voor de uitvoering; (centrale) privacy officers en is hier ruimte voor?

>     3) De FG kijkt naar de wet en interpreteert dit. Zorg dat deze interpretatie gelijk is binnen de organisatie en de collega’s die binnen de governance structuur samenwerken.

Reviewer: Een FG neemt meerdere zaken mee in zijn of haar beoordeling. Dit is daarom onjuist.

>     3) De FG kijkt naar de wet en interpreteert dit. Zorg dat deze interpretatie gelijk is binnen de organisatie en de collega’s die binnen de governance structuur samenwerken.

Reviewer: ```suggestion
    3) Zorg dat interpretaties over wet- en regelgeving gelijk is binnen de organisatie en de collega’s die binnen de governance structuur samenwerken.
```

>     4) Er is beleid gemaakt, maar te weinig bezetting en/of andere prioriteiten. Zorg voor een realistisch beleid en daarop een realistische planning. We willen vaak te veel (de gehele wetgeving meteen beheersen).

Reviewer: ```suggestion
    4) Er is beleid gemaakt, maar in de praktijk is te weinig capaciteit om dit uit te voeren. Stel een realistisch beleidsplan op om de gestelde doelen daadwerkelijk te kunnen realiseren. 
```
